pkg/auth/service.go

```diff
@@ -778,152 +753,148 @@ func (s *DBAuthService) AddCredentials(ctx context.Context, username, accessKeyI
 	if err != nil {
 		return nil, err
 	}
-	credentials, err := s.db.Transact(ctx, func(tx db.Tx) (interface{}, error) {
-		user, err := getUser(tx, username)
-		if err != nil {
-			return nil, err
-		}
-		c := &model.Credential{
-			BaseCredential: model.BaseCredential{
-				AccessKeyID:                   accessKeyID,
-				SecretAccessKey:               secretAccessKey,
-				SecretAccessKeyEncryptedBytes: encryptedKey,
-				IssuedDate:                    now,
-			},
-			UserID: user.ID,
-		}
+	user, err := s.GetUser(ctx, username)
+	if err != nil {
+		return nil, fmt.Errorf("%s: %w", username, err)
```

Comment:
wrong formatting

Reply:
I don't understand why?

Reply:
Basically because go errors are not debug log and extending this behaviour throughout the different layers will result with errors like: `<username>: <username>: <username>: <username>: some db error`